README.md

```diff
@@ -54,12 +56,30 @@ See the [documentation](https://soundata.readthedocs.io/) for more examples and
 We welcome and encourage contributions to this library, especially new dataset loaders. Please see [contributing](https://soundata.readthedocs.io/en/latest/source/contributing.html) for guidelines. Feel free to [open an issue](https://github.com/soundata/soundata/issues) if you have any doubt or your run into problems when working on the library.
 
 
-### Citing
+### Releases
 
+The Soundata Zenodo repository is the preferred source for downloading the software releases.
 
-```
-TBA
-```
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.11518021.svg)](https://doi.org/10.5281/zenodo.11518021)
 
 
-When working with datasets, please cite the version of `soundata` that you are using **AND** include the reference of the dataset, which can be found in the respective dataset loader using the `cite()` method. 
+### Citing
+
+If you use Soundata please cite the JOSS article **AND** indicate the version used with the corresponding DOI of the version release in Zenodo (`10.5281/zenodo.11580085` for v1.0.1, for example).
```

Comment:
this **AND** should change, right? They should cite the software with the DOI if using soundata in the pipeline, and citing the paper if they use it as inspiration for something similar, right?